ofrak_core/ofrak/core/filesystem.py

```diff
@@ -22,6 +26,14 @@
 )
 
 
+def _warn_chmod_chown_windows():  # pragma: no cover
+    warnings.warn(
+        f"os.chown and os.chmod do not work on Windows platforms. \
+        Unix-like file ownership and permissions will not be properly handled while using OFRAK on this platform. \
+        If you require extended attributes, please use a different platform."
+    )
```

Comment:
if this was a logger warning, flush to disk would generate a whole stream of duplicate messages for each file. the warning module doesn't repeat them

Reply:
Seems reasonable to me. Worth adding a comment saying that, though. That way someone doesn't come along and refactor this to use `logging` in the future without context for why this doesn't use it already.

Reply:
We do `logging.captureWarnings` in OFRAK Contexts, so the message is spit out by logging anyway. This is the kind of message that should be a `warnings.warn`

Reply:
and also the one in xattr_stub

Reply:
If you want to open an issue for changing other uses of `logging.warning` to `warnings.warn`, we can resolve this thread. Seems out of scope for this PR, though. 

In general, it is helpful to add code comments describing "why" we're doing stuff when it defies convention in the codebase. I agree that using `warnings.warn` is reasonable here in the context of this PR. But in the context of the rest of OFRAK, it will likely not be clear to a future reader. If we open an issue for a future PR to change the usage at the spots you indicate (and move towards changing the convention in the codebase), I think that is a suitable alternative to adding one-off documentation here.